docs/secure-connections/sauce-connect/setup-configuration/sc-docker.md

```diff
@@ -0,0 +1,121 @@
+---
+id: sc-docker
```

Comment:
I'd prefer to name it "docker" as "Sauce Connect" is already in the URL and it'd look cleaner

```suggestion
id: docker
```